BackendSector/databaseMiddleware.js

```diff
@@ -0,0 +1,54 @@
+let school = require("./databaseSchool");
+
+const addClass = (req, res, next) => {
+    try {
+        let newClass = school.addClass(req.body.name, req.body.teacher)
+        debugger
```

Comment:
debugger works on the frontend

Reply:
Yeah it wasn't working. Forgot to remove, removed.